packages/react-table/src/components/Table/Table.tsx

```diff
@@ -61,6 +61,8 @@ export interface TableProps extends OUIAProps {
   onSelect?: OnSelect;
   /** Enables or Disables the ability to select all - this is mutually exclusive with radio button select variant */
   canSelectAll?: boolean;
+  /** Enables or Disables the ability to expand all */
```

Comment:
nit.  Can this be sentence case.

Reply:
sure, i just went based on the previous (canSelectAll) property.. i can change both

Reply:
Thank you!